### DISCUSSION TITLE
[Bug] Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'getTimeInfo')

### DISCUSSION THREAD

**Discussion by wangchao732** [apache/skywalking#12170] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

UI (apache/skywalking-booster-ui)

### What happened

skywalking-helm  helm install

![image](https://github.com/apache/skywalking/assets/19440083/cb799e4b-6828-473d-b2e7-162cad556170)

### What you expected to happen

![image](https://github.com/apache/skywalking/assets/19440083/703b1fc8-cf95-4635-93bd-a3bd1ab9b1c7)

### How to reproduce

/

### Are you willing to submit a pull request to fix on your own?

- [X] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
This usually means your OAP server is not up in healthy. Check your logs.
